admin/InitDb.pl

```diff
@@ -356,22 +356,18 @@ sub CreateRelations
             RunSQLScript($DB, 'wikidocs/CreateReplicationTriggers.sql', 'Creating wikidocs replication triggers ...');
         }
 
-        if ($dbmirror2) {
```

Comment:
They will still use the old `dbmirror` tables for replication, but as part of this schema change I wanted to make sure that the (as of yet unused) dbmirror2 schema is created, so that we could seamlessly switch to it once ready.